app/cli/cmd/workflow_create.go

```diff
@@ -64,6 +65,15 @@ func newWorkflowCreateCmd() *cobra.Command {
 				return err
 			}
 
+			// If the robot account creation is not skipped via --skip-robot-account-create , create a new one
+			if !skipRACreate {
+				resRA, errRA := action.NewWorkflowRobotAccountCreate(actionOpts).Run(res.ID, "default")
+				if errRA != nil {
+					return errRA
+				}
+				res.RobotAccountID = resRA.ID
+				res.RobotAccountKey = resRA.Key
+			}
 			return encodeOutput([]*action.WorkflowItem{res}, WorkflowListTableOutput)
```

Comment:
See my comment above of why I think adding extra properties to the struct might not be the best way, what about rendering the robot account here instead using the already defined output?

something like

```diff
-                       return encodeOutput([]*action.WorkflowItem{res}, WorkflowListTableOutput)
+                       if err := encodeOutput([]*action.WorkflowItem{res}, WorkflowListTableOutput); err != nil {
+                               return err
+                       }
+
+                       // Check that we are rendering the table so we don't break the json output
+                       if flagOutputFormat == formatTable {
+                               return encodeOutput([]*action.WorkflowRobotAccountItem{robotAccount}, robotAccountListTableOutput)
+                       }
+
+                       return nil
                },

```